geomesa-tools/pom.xml

```diff
@@ -125,6 +125,10 @@
             <artifactId>jcommander</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.geotools</groupId>
+            <artifactId>gt-epsg-hsql</artifactId>
+        </dependency>
+        <dependency>
```

Comment:
it was transitively included before - we had to explicitly include it after the changes.